### ISSUE TITLE
Concepção do Sistema

### ISSUE THREAD

**Issue by phmiranda** [JavaBrasil/documentacao-helpdesk#1]

**Comment by luiztoni** [comment]
Podemos elaborar um documento aberto no Google Docs ou Office Online e deixar um link editável aqui. Enquanto isso vamos pensando nas funcionalidades básicas. O que acham?

**Comment by phmiranda** [comment]
@luiztoni concordo totalmente, eu abri essa issue, mas estou pensando em um padrão de nome para facilitar na busca, como um espécie de número da requisição + objetivo da issue resumida. Nessa issue que abri pretendo listar as funcionalidade iniciais do projeto através dos casos de usos listados no repositório (UC001, UC002 e UC003) no diretório de caso de uso.

**Comment by CaiqueJhones** [comment]
@luiztoni Os docs estão offline aqui no repositório, não achar melhor assim, pois poderemos ter um maior controle quanto ao versionamento utilizando o git?

**Comment by luiztoni** [comment]
Entendi, é uma ideia boa, também fica bem organizado mante-los separados aqui no repositório, até por ser um repositório pra documentação. Vi agora também que tem alguns arquivos de modelagem de sistema e de banco de dados, depois se possível vamos deixar o link dos programas usados na edição e criação destes arquivos e uma versão exportada pra imagem png atualizada a cada alteração, assim fica mais fácil para todos acompanharem.

**Comment by phmiranda** [comment]
Acharia melhor @CaiqueJhones disponibilizar a última versão em produção do documento online no formato PDF, mas os documentos editáveis serem mantidos versionados corretamente no repositório.

**Comment by phmiranda** [comment]
@luiztoni justamente isso que pensei, as extrações da modelagem do sistema serem disponibilizadas online, mas os documentos em si serem mantidos apenas no repositório para versionamento (observação: eu estou procurando os templates do MDS e vou implantar em todos os documentos e assim manter um padrão, vocês teriam todo o design do grupo para colocar nos documentos ?)

**Comment by luiztoni** [comment]
O que seria MDS? bem, seria interessante, tanto pra padronizar como para manter uma identidade visual, que aliás o grupo ainda não tem, estamos começando a pensar em criar

**Comment by phmiranda** [comment]
MDS - Metodologia de Desenvolvimento de Software

**Comment by phmiranda** [comment]
Consegui encontrar os artefatos que utilizava no datasus, esses documentos vão auxiliar a comunidade no processo de desenvolvimento e documentação dos sistemas, consequentemente não iremos usar todos os artefatos e todo o processo deles, mas podemos adaptar ele a nossa realidade para executar o levantamento de requisitos, codificação e executar os testes unitários, automatizados e funcionais. Estou disponibilizando o link do repositório [](http://datasus.saude.gov.br/menu-pag-sobre-a-mds)

**Comment by luiztoni** [comment]
Entendi, podemos usar metodologias mais tradicionais e mesclar com aspectos de Agile como Kanban ou Scrum de acordo a conveniência, por mim sem problemas
